external-crates/move/move-compiler/src/naming/translate.rs

```diff
@@ -62,7 +64,8 @@ struct ResolvedModuleFunction {
 struct Context<'env> {
     env: &'env mut CompilationEnv,
     current_module: Option<ModuleIdent>,
-    scoped_types: BTreeMap<ModuleIdent, BTreeMap<Symbol, (Loc, ModuleIdent, AbilitySet, usize)>>,
+    scoped_types:
+        BTreeMap<ModuleIdent, BTreeMap<Symbol, (Loc, ModuleIdent, AbilitySet, usize, bool)>>,
```

Comment:
Might be time to make a struct here :) `usize, bool` isn't particularly illuminating